lib/route.js

```diff
@@ -231,6 +234,15 @@ function buildRouting (options) {
         return
       }
 
+      // TODO: @MetCoder95 Implement hability for ignoring paths
```

Comment:
This is not on the principles of Fastify. There is no ignoring (and regexps, etc). We can support it in an ecapsulation-friendly way if we want to.

Reply:
Sure, my bad! Actually, it's a leftover as I followed the suggestion of using a single-route/global flag to enable/disable the feature